### PR TITLE
Add method getEntityUsedForUserResponse in templateGateway (#1645)

### LINKED ISSUE #1645: Add a method for template to get the entity used for response
Add a method getEntityUsedForUserResponse in templateGateway 

This method returns the entity that will be used when the user will send the response

### PR DESCRIPTION
**Pull request by quinarygio**
Fix #1645 

Release notes:

In Features section:
#1645 Add a method for template to get the entity used for response

Signed-off-by: Giovanni Ferrari <giovanni.ferrari@soft.it>

### CODE REVIEW

File: src/test/resources/bundles/userCardExamples2/template/en/question.handlebars [hunk 1]
```diff
@@ -69,6 +69,14 @@
     }
     loadEntitiesList();
 
+    function loadEntityUsedForResponse() {
+        let responseEntityDiv = document.getElementById("response-entity-div");
+        let responseEntity = templateGateway.getEntityUsedForUserResponse();
+        let userAllowedToRespond = templateGateway.isUserAllowedToRespond();
+        if (userAllowedToRespond && responseEntity) responseEntityDiv.innerHTML = "Entity used by user to respond : " + responseEntity;
+    }
```
Comment (freddidierRTE) [inline src/test/resources/bundles/userCardExamples2/template/en/question.handlebars:77]: Show the name of the entity as for entites allowed to respond (use templateGateway.getEntityName)

File: src/test/resources/bundles/userCardExamples2/template/fr/question.handlebars [hunk 2]
```diff
@@ -68,6 +70,13 @@
     }
     loadEntitiesList();
 
+    function loadEntityUsedForResponse() {
+        let responseEntityDiv = document.getElementById("response-entity-div");
+        let responseEntity = templateGateway.getEntityUsedForUserResponse();
+        let userAllowedToRespond = templateGateway.isUserAllowedToRespond();
+        if (userAllowedToRespond && responseEntity) responseEntityDiv.innerHTML = "Entité utilisée par l'utilisateur pour répondre : " + responseEntity;
```
Comment (freddidierRTE) [inline src/test/resources/bundles/userCardExamples2/template/fr/question.handlebars:77]: Show the name of the entity as for entites allowed to respond (use templateGateway.getEntityName)

File: src/test/resources/bundles/userCardExamples2/template/en/question.handlebars (hunk 1 above)
Comment (quinarygio) [reply src/test/resources/bundles/userCardExamples2/template/en/question.handlebars]: Done

File: src/test/resources/bundles/userCardExamples2/template/fr/question.handlebars (hunk 2 above)
Comment (quinarygio) [reply src/test/resources/bundles/userCardExamples2/template/fr/question.handlebars]: Done

**Pull request merged**